bin/save-ebuild-env.sh

```diff
@@ -92,6 +92,12 @@ __save_ebuild_env() {
 	___eapi_has_in_iuse && unset -f in_iuse
 	___eapi_has_version_functions && unset -f ver_cut ver_rs ver_test
 
+	if ___eapi_has_pipestatus; then
+		unset -f pipestatus
+	else
+		unset -f assert
+	fi
```

Comment:
... and it is not needed here:
```suggestion
	___eapi_has_pipestatus && unset -f pipestatus
```